opencti-platform/opencti-graphql/src/modules/attributes/stixObject-registrationAttributes.ts

```diff
@@ -1,11 +1,9 @@
-import { type AttributeDefinition, created, files, modified, xOpenctiStixIds } from '../../schema/attribute-definition';
+import { type AttributeDefinition, files, xOpenctiStixIds } from '../../schema/attribute-definition';
 import { schemaAttributesDefinition } from '../../schema/schema-attributes';
 import { ABSTRACT_STIX_OBJECT } from '../../schema/general';
 
 const stixObjectAttributes: Array<AttributeDefinition> = [
   xOpenctiStixIds,
-  created,
-  modified,
```

Comment:
For my own curiosity, why do we remove those fields from `stixObjects`?

Reply:
created and modified are only field for stix core objects and stix meta objects, not for observables.
So I removed them from stixObjects , to only put them for stix core objects and stix meta objects.

Reply:
they're moved in stixDomainObjects apparently.
But that's not the model definition documented... so I'm curious too :)